SUMMARY.md

```diff
@@ -0,0 +1,28 @@
+## Table of Contents
+
+* [Connecting to Database](content/ContectingToDatabase/ConnectingToDatabase.q.md) ** - minutes**
+    > Learn how you can use a specific class or command to establish a connection between PHP and PostgreSQL.
+
+* [PHP Data Objects](content/PDOClasses/PDOClasses.q.md) ** - minutes**
+    > In this lesson, you will learn about PHP Data Objects (`PDO`) classes and how to execute queries using `PDO::query()`.
+
+* [Prepared Statement](content/PreparedStatement/PreparedStatement.q.md) ** - minutes**
+    > In this lesson, you will learn the basics of using `PDO::prepare()` with its parameters in PHP.
+
+* [Prepared Statement with execute()](content/UsingExecute/UsingExecute.q.md) ** - minutes**
+    > Discover how you can execute prepared statements using `PDOStatement::execute()` with binding methods such as `bindParam()` and `bindValue()`.
+
+* [Prepared Statement with fetch()](content/UsingFetch/UsingFetch.q.md) ** - minutes**
+    > Learn how you can use `PDOStatement::fetch()` to fetch data and return a specific row of data from a result set.
+
+* [Prepared Statement with fetchColumn()](content/UsingFetchColumn/UsingFetchColumn.q.md) ** - minutes**
+    > Learn how you can use `PDOStatement::fetchColumn()` to return one column field from the next row of a result set.
+
+* [Prepared Statement with fetchAll()](content/UsingFetchAll/UsingFetchAll.q.md) ** -minutes**
+    > Discover how you can use `PDOStatement::fetchAll()` to return an array that holds all the rows in the result set.
+
+* [PDO Stored Procedures](content/PDOStoredProcedures/PDOStoredProcedures.q.md) ** - minutes**
+    > Explore the things you can do with calling stored procedures in PDO such as `PDOStatement::columnCount()`, `PDOStatement::rowCount()`, and `PDOStatement::nextRowSet()`.
```

Comment:
This statement is not true since `columnCount()` and the rest of the `PDOStatement` methods mentioned here are not stored procedures. Please check this definition below:
>A stored procedure is a prepared SQL code that you can save, so the code can be reused over and over again.

I think we can still retain this lesson about `stored procedure` but make sure to create a separate lesson about the `PDOStatement` methods such as `rowCount()`, `columnCount()`, and `nextRowSet()`. Thanks.